### PR TITLE
coreJSON ReadMe update

### PR DESCRIPTION
**Pull request by yngki**
ReadMe Update for CoreJSON

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## coreJSON Library
 
-A parser strictly enforcing the ECMA-404 JSON standard, suitable for microcontrollers
+This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and is suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
```
Comment (tanmoysen) [inline README.md:3]: This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for low memory footprint embedded devices. The coreJSON library is distributed under the [MIT open souce license](LICENSE)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## coreJSON Library
 
-This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
+This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and is suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
```
Comment (tanmoysen) [inline README.md]: Can we change the language? "This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for constrained embedded devices. It is distributed under the [MIT open source license](LICENSE)

Comment (yngki) [reply README.md]: (this has already been made, tanmoy was viewing the old file)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ## coreJSON Library
 
-A parser strictly enforcing the ECMA-404 JSON standard, suitable for microcontrollers
+This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for low memory footprint embedded devices. The coreJSON library is distributed under the [MIT Open Source License](LICENSE).
+
+
+This library has gone through code quality checks including for [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html), [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx), [Coverity statical analysis](https://scan.coverity.com/), and [AWS CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721) to validate memory safety, thread safety and functional correctness proof.  Deviations to the MISRA C:2012 guidelines are documented under [MISRA Deviations](MISRA.md).
```
Comment (dan4thewin) [inline README.md:6]: Some grammar issues here, as well as conflating tools used with what is being measured.  Maybe something like:

"This library is reported free from errors using both static analysis (Coverity) and formal verification (CBMC).  The library maintains a low complexity score throughout (reported by GNU Complexity).  The library complies with all mandatory MISRA 2012 rules and almost all non-mandatory rules.  Deviations..."

**Review by dan4thewin — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 ## coreJSON Library
 
-A parser strictly enforcing the ECMA-404 JSON standard, suitable for microcontrollers
+This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for low memory footprint embedded devices. The coreJSON library is distributed under the [MIT Open Source License](LICENSE).
```
[the remark this answers is not in the document]
Comment (yngki) [reply README.md]: howdy, leaving as is.. this is the language used on our marketing pages as well ^ 

have changed language to "strictly enforced".

File: README.md (hunk 3 above)
Comment (yngki) [reply README.md]: worked with Tanmoy and have updated the language

**Review by dan4thewin — APPROVED** [review]

**Review by abhidixi11 — APPROVED** [review]

**Pull request merged**